### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by flattenstream**
Just adding the Korean language. Thanks!

### CODE REVIEW

File: lang/ko.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "CN.ConcentrationAbbr": "Conc",
+  "CN.ConfirmEndConcentrationText": "{name}의 집중이 끝나게 될 것입니다.",
+  "CN.ConfirmEndConcentrationTitle": "{name}의 집중을 끝내시겠습니까?",
+  "CN.DisplayItem": "아이템 표시",
+  "CN.FlagConcentrationAbility": "집중 능력",
+  "CN.FlagConcentrationAbilityHint": "이 캐릭터가 집중을 유지하는데 사용하는 능력입니다.",
+  "CN.FlagConcentrationAdvantage": "집중 유리함",
+  "CN.FlagConcentrationAdvantageHint": "이 캐릭터는 집중을 유지하는 굴림에 유리합니다.",
+  "CN.FlagConcentrationBonus": "집중 보너스",
+  "CN.FlagConcentrationBonusHint": "집중을 유지하는 내성 굴림에 받는 보너스입니다.A bonus to saving throws to maintain concentration. This field supports dynamic values such as @classes.sorcerer.levels as well as dice expressions.",
```
Comment (krbz999) [inline lang/ko.json:19]: Was this missed? :)

File: lang/ko.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+  "CN.AbilityDialogWarningItem": "이 물건이나 특성을 사용하면 {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningSpell": "이 주문을 시전하면, {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningSpellLevel": "이 주문을 {level} 외의 레벨로 시전하면, {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningUnfocusedItem": "이 물건이나 특성을 사용할 수 있지만, 그것에 집중할수는 없습니다.",
+  "CN.AbilityDialogWarningUnfocusedSpell": "이 주문을 시전할 수 있지만, 그것에 집중할수는 없습니다.",
+  "CN.ButtonEndConcentration": "집중 끝내기",
+  "CN.ButtonRemoveTemplates": "템플릿 제거",
+  "CN.ButtonSavingThrow": "내성굴림DC {dc} {saveType}",
+  "CN.ConcentrationAbbr": "Conc",
```
Comment (krbz999) [inline lang/ko.json:10]: I'm not familiar with the language, so I can't speak to how abbreviations work. Is it correct to not translate abbreviations?

**Review by krbz999 — CHANGES_REQUESTED** [review]
Looks good to me, I just have a few questions in the review below.

**Comment by flattenstream** [comment]
Oh, I missed a few. I just fixed them, Thank you!

**Review by krbz999 — APPROVED** [review]

**Comment by krbz999** [comment]
Thank you for this. I'm always happy to see more localization added.

**Pull request merged**